frontend/src/__tests__/cypress/cypress/utils/mockUsers.ts

```diff
@@ -108,38 +108,189 @@ const setUserConfig = (userConfig: UserConfig = {}, isAllowed = true) => {
+        displayName: 'Self Provisioner:',
+        message: 'Grant access to project creation',
+        consoleProps: () => {
+          return {
+            Verb: resourceAttributes.verb,
+            Resource: resourceAttributes.resource,
+            Namespace: resourceAttributes.namespace,
+            Group: resourceAttributes.group,
+          };
+        },
+      });
+      return true;
+    }
+
+    if (resource === 'rolebindings' && !projectAdmin) {
+      Cypress.log({
+        name: 'SSAR Permission Check',
+        displayName: 'Project Users:',
+        message: 'Access denied (requires Project Admin)',
+        consoleProps: () => {
+          return {
+            Verb: resourceAttributes.verb,
+            Resource: resourceAttributes.resource,
+            Namespace: resourceAttributes.namespace,
+            Group: resourceAttributes.group,
+          };
+        },
+      });
+      return false;
+    }
+
+    if (PROJECT_ADMIN_RESOURCES.includes(resource) && EDIT_VERBS.includes(verb)) {
+      if (projectAdmin) {
+        Cypress.log({
+          name: 'SSAR Permission Check',
+          displayName: 'Project Admin:',
+          message: 'Edit access granted',
+          consoleProps: () => {
+            return {
+              Verb: resourceAttributes.verb,
+              Resource: resourceAttributes.resource,
+              Namespace: resourceAttributes.namespace,
+              Group: resourceAttributes.group,
+            };
+          },
+        });
+        return true;
+      }
+      Cypress.log({
+        name: 'SSAR Permission Check',
+        displayName: 'Project Users:',
+        message: 'Edit access denied',
+        consoleProps: () => {
+          return {
+            Verb: resourceAttributes.verb,
+            Resource: resourceAttributes.resource,
+            Namespace: resourceAttributes.namespace,
+            Group: resourceAttributes.group,
+          };
+        },
+      });
+      return false;
+    }
+    if (namespace) {
+      if (namespace === 'opendatahub' && !productAdmin) {
+        Cypress.log({
+          name: 'SSAR Permission Check',
+          displayName: 'Permission:',
+          message: 'Access denied to opendatahub namespace',
+          consoleProps: () => {
+            return {
+              Verb: resourceAttributes.verb,
+              Resource: resourceAttributes.resource,
+              Namespace: resourceAttributes.namespace,
+              Group: resourceAttributes.group,
+            };
+          },
+        });
+        return false;
+      }
+      Cypress.log({
+        name: 'SSAR Permission Check',
+        displayName: 'Permission:',
+        message: 'Access granted within namespace',
+        consoleProps: () => {
+          return {
+            Verb: resourceAttributes.verb,
+            Resource: resourceAttributes.resource,
+            Namespace: resourceAttributes.namespace,
+            Group: resourceAttributes.group,
+          };
```

Comment:
Curious if you looked at the whole object of `resourceAttributes` -- I wonder if we could not just use the object as-is. There are other properties that could be used (`resourceName` is a known one that would be very valuable in a project case)